conf/context.go

```diff
@@ -0,0 +1,181 @@
+package conf
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/BurntSushi/toml"
+)
+
+type audiusContextKey string
+
+const (
+	ConfigVersion = "0.1"
+	ContextKey    = audiusContextKey("audius_cobra_context")
+)
+
+func ReadOrCreateContextConfig() (*ContextConfig, error) {
+	confDir, err := getConfigBaseDir()
+	if err != nil {
+		return nil, err
```

Comment:
TODO: improve error handling to be consistent and more descriptive throughout the program